src/main/java/org/opensearch/sdk/ExtensionRestHandler.java

```diff
@@ -9,6 +9,7 @@
 
 import java.util.List;
 
+import org.opensearch.extensions.rest.ExtensionRestRequest;
```

Comment:
Thinking out loud. Why are we keeping `ExtensionRestRequest` in OpenSearch and not in SDK?

Reply:
Because the previous `RestSendToExtensionRequest` entirely duplicated it.  Literally all the lines were the same except for a few naming differences.

Reply:
Also it makes sense (see #165) to move the ExtensionRestResponse there as well since it's really the same thing as the BytesRestResponse with some tweaks.

Reply:
This clears my questions. Just one thing to add can we keep the name as `RestSendToExtensionRequest` instead of `ExtensionRestRequest` for better understanding. Can be address in the next PR

Reply:
Let’s discuss on #165, because both classes are part of the API we expect extension authors to use.  The ExtensionRestRequest is the argument type for the handleRequest method and it closely parallels the same method on plugins and I’d really like similar names.